ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent.json

```diff
@@ -1033,6 +1033,20 @@
     "UserRole": "caseworker-publiclaw-magistrate",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createNoticeOfProceedings",
+    "UserRole": "caseworker-publiclaw-gatekeeper",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createNoticeOfProceedings",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "R"
+  },
```

Comment:
probably should go into DEPRECATED file

Reply:
would in this case have the two rules (one with  "CRUD": "R" and  the other with "CRUD": "CRU") transformed?

Reply:
both should have R